### PR TITLE
Added the buyings strategy document

### PR DESCRIPTION
**Pull request by daanwerf**
The document for the buying strategy. If I missed anything or need to improve on somethings please tell. This document is part of the user story:

"As an agent When in a 
simulation I am able to buy 
ground if I think this is 
benificial for me. To construct 
things that are part of my goals.
"

### CODE REVIEW

File: documents/Strategies/Buying strategy.tex [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+\section{When/why to buy ground}
+The final goal of the agent as a stakeholder is to make sure that all its indicators are as high as possible. This means for the agent that portrays the TU Delft that it wants to have education buildings that are as modern, space efficient and green as possible. This means that if the agent owns buildings that are (too) old, it will try to demolish these buildings and build better new buildings. If the agent does not own the land, he needs to buy it first in order to be able to use it to raise its indicators. 
+
+
+\section{Which land to buy}
+For the agent to decide which land to buy depends on a few factors. First of all, because one of his indicators is to keep a positive balance, the agent has to take the price of the land into accountl. Secondly, the building that the agent plans on constructing on the land needs to have enough space to be build. This means that the land that the agent wants to buy needs to be large enough to build the desired building. Based on both of these factors the agent will decide which land to buy.
```
Comment (spiderbiggen) [inline documents/Strategies/Buying strategy.tex]: > the agent has to take the price of the land into **accountl**

accountl should be account

**Comment by spiderbiggen** [comment]
This looks good apart from a minor spelling mistake

File: documents/Strategies/Buying strategy.tex [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+\section{When/why to buy ground}
+The final goal of the agent as a stakeholder is to make sure that all its indicators are as high as possible. This means for the agent that portrays the TU Delft that it wants to have education buildings that are as modern, space efficient and green as possible. This means that if the agent owns buildings that are (too) old, it will try to demolish these buildings and build better new buildings. If the agent does not own the land, he needs to buy it first in order to be able to use it to raise its indicators. 
+
+
+\section{Which land to buy}
+For the agent to decide which land to buy depends on a few factors. First of all, because one of his indicators is to keep a positive balance, the agent has to take the price of the land into accountl. Secondly, the building that the agent plans on constructing on the land needs to have enough space to be build. This means that the land that the agent wants to buy needs to be large enough to build the desired building. Based on both of these factors the agent will decide which land to buy. 
+
```
Comment (jzoon) [inline documents/Strategies/Buying strategy.tex:32]: Delete the '1' after account.

Also, it isn't really clear which land will be bought by the agent. Shouldn't it be close to the TU Delft, but not in the middle or something. Maybe explain the location a bit more.

**Comment by jzoon** [comment]
Was fixed, merged it

**Pull request merged**